shell-completion/bash/systemd-analyze

```diff
@@ -211,6 +212,13 @@ _systemd_analyze() {
         else
             comps=$( __get_architectures )
         fi
+
+    elif __contains_word "$verb" ${VERBS[FDSTORE]}; then
+        if [[ $cur = -* ]]; then
+            comps='--help --version --system --user --global -H --host -M --machine --no-pager --json=off --json=pretty --json=short --root --image'
```

Comment:
Maybe better to suggest json option values in a separated way, like we do in resolvectl.

Reply:
I think it might complicate a lot this file. I've done it similarly to previous completion in same file (look at code above my change).
If I try to add split, I might result in accidentally suggesting values for `--json` for subcommands which don't have JSON at all.
What do you think?

Reply:
OK, let's keep this and existing code now, and revisit that later if necessary.